src/eva/core/data/datasets/embeddings/base.py

```diff
@@ -54,12 +56,38 @@ def __init__(
         self._root = root
         self._manifest_file = manifest_file
         self._split = split
-        self._column_mapping = self.default_column_mapping | column_mapping
+        self._column_mapping = column_mapping
```

Comment:
hmmm why do we change this? here the `self.default_column_mapping | column_mapping`, allows the user can to alter only one and the rest to remain the same (so he dont have to repeat the fields if they are the same as defaults), so now I think we have to repeat all of them

Reply:
good catch! I reverted this change now.